incubator/busbar/templates/daemonset.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "kubeRegistry.fullname" . }}-internal
+  labels:
+    app: {{ template "kubeRegistry.fullname" . }}-internal
```

Comment:
The `app` label should be `{{ template "busbar.name" . }}`. Please fix this throughout the chart.

Reply:
Hmm... but this daemonset is not related to busbar. It's the kubernetes registry one.

Maybe it should be `kubeRegistry.name` instead `fullname`?